lib/src/actions.dart

```diff
@@ -123,6 +135,8 @@ class SlidableAction extends StatelessWidget {
     this.icon,
     this.spacing = 4,
     this.label,
+    this.shape = const RoundedRectangleBorder(),
```

Comment:
By following the previous comment, we dont' need to redefine them here.

Reply:
fixed